### PR TITLE
Helm celery main secrets

### PR DESCRIPTION
**Pull request by P0NDER0SA**
## What are you changing?

Writing helm/helmfile code for celery that will deploy (on dev for now) and reads all secrets from the AWS Secrets Manager.

This code will be much more streamlined and easier to manage than the mountains of kustomize code used previously.

## Provide some background on the changes
https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/457

- [ ] I have verified that the tests / deployment actions succeeded
- [ ] I have verified that any affected pods were restarted successfully
- [ ] I have verified that I can still log into [Notify production](https://notification.canada.ca)
- [ ] I have verified that the smoke tests still pass on production
- [ ] I have communicated the release in the #notify Slack channel.

### CODE REVIEW

File: helmfile/charts/notify-celery/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+{{- range $node := .Values.nodes.primaryAndScalable }}
```
Comment (ben851) [inline helmfile/charts/notify-celery/templates/deployment.yaml:1]: Ah I see - nice!

**Review by ben851 — APPROVED** [review]
Approved but can we set the configuration for min/max pods for the HPA?

**Comment by P0NDER0SA** [comment]
> Approved but can we set the configuration for min/max pods for the HPA?

Ah, that highlights something i need to update... I'll get that fixed.  and to answer the question, yes

File: helmfile/charts/notify-celery/templates/deployment.yaml (hunk 1 above)
Comment (P0NDER0SA) [reply helmfile/charts/notify-celery/templates/deployment.yaml:1]: I'm pushing those configs to the nodes section so they can be changed if you like :)

**Pull request merged**